dotnet/Face/FaceQuickstart.cs

```diff
@@ -97,9 +97,9 @@ static void Main(string[] args)
             LargePersonGroup(client, IMAGE_BASE_URL, RECOGNITION_MODEL4).Wait();
             // Group faces - automatically group similar faces.
             Group(client, IMAGE_BASE_URL, RECOGNITION_MODEL4).Wait();
-            // FaceList - create a face list, then get data
             // </snippet_maincalls>
 
+            // FaceList - create a face list, then get data

```

Comment:
Here I move the comment line "// FaceList - ... " to be outside the code snippet in the quickstart doc